src/python/python.cpp

```diff
@@ -250,6 +243,9 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
         return false;
 #endif
   });
+
+  m.def("set_current_gpu_device_id", [](int device_id) { Ort::SetCurrentGpuDeviceId(device_id); });
+  m.def("get_current_gpu_device_id", []() { return Ort::GetCurrentGpuDeviceId(); });
```

Comment:
Do we not need this in other language bindings?

Reply:
We should probably have it. I'll add it to the C# bindings.

Reply:
Done